### ISSUE TITLE
Update accessibility statement to mention issues caused by Tech Docs Template

### ISSUE THREAD

**Issue by EoinShaughnessy** [alphagov/govuk-design-system#1770]
## What
We need to update our accessibility statement to:

- say we're aware of accessibility issues caused by the Technical Documentation Template (TDT)
- link to the [TDT accessibility statement](https://tdt-documentation.london.cloudapps.digital/accessibility/#using-the-technical-documentation-template-for-your-own-documentation), where users can read about the issues in detail

## Why
We recently updated Frontend's TDT to use the latest version of the tech docs gem. 

This latest version incorporated the accessibility fixes identified in a Digital Accessibility Centre (DAC) audit from September 2020.

However, in April 2020, a DAC audit of GovWifi tech docs found more accessibility issues with the TDT.

There are 4 WCAG-related issues and 2 non WCAG-related issues. GovWifi will try to fix all 6 issues by the end of July, although they are prioritising the WCAG issues.

The WCAG issues are:

- In some circumstances, the main page heading is automatically in focus when the page loads. This means keyboard-only users may miss the navigation elements. This fails [WCAG 2.1 success criterion 2.4.3 - Focus Order](https://www.w3.org/TR/WCAG21/#focus-order).
- The navigation elements are not marked up semantically as hierarchical headings, which means moving through the navigation using a screen reader may be more difficult. This fails [WCAG 2.1 success criterion 1.3.1 - Info and Relationships](https://www.w3.org/TR/WCAG21/#info-and-relationships).
- There are sections of content with show / hide functionality. The `aria-expanded` attribute is not implemented correctly, so users may miss that menu items are expandable. This fails [WCAG 2.1 success criterion 4.1.2 - Name, Role, Value](https://www.w3.org/TR/WCAG21/#name-role-value).
- When a user searches, the search results dialogue hides the main page region from assistive technology. This fails [WCAG 2.1 success criterion 3.2.2 - On Input](https://www.w3.org/TR/WCAG21/#on-input) and [WCAG 2.1 success criterion 2.4.3 - Focus Order](https://www.w3.org/TR/WCAG21/#focus-order).

## Who needs to know about this
Developers, Technical Writer

## Done when
- [ ] Technical writer raises PR with new content
- [ ] Developer reviews PR
- [ ] PR clears pre-i and 2i
